components/pages/Story/layouts/default/styles/body/Story.body.qa.ts

```diff
@@ -19,7 +19,7 @@ export const storyBodyQAStyles = (theme: Theme) =>
         theme.palette.primary.main
       }`,
 
-      '& h3': {
+      '& h2': {
         margin: '1rem 0 1rem',
```

Comment:
This will break styling of QA's in body content, unless you have already changed the markup being generated on Drupal's end.

Reply:
Ah, I have not changed that markup. I'll revert this and file a Drupal ticket.

Reply:
Done!